src/main/java/players/mcts/MCTSPlayer.java

```diff
@@ -247,8 +247,15 @@ protected void createRootNode(AbstractGameState gameState) {
         if (newRoot == null) {
             if (getParameters().opponentTreePolicy == MultiTree)
                 root = new MultiTreeNode(this, gameState, rnd);
-            else
-                root = SingleTreeNode.createRootNode(this, gameState, rnd, getFactory());
+            else {
+                if(getParameters().information == MCTSEnums.Information.Perfect_Information)
```

Comment:
Also, this doesn't take account of reuseTree (not the end of the world, but I wonder if we could implement this)